util/template.go

```diff
@@ -0,0 +1,28 @@
+package util
+
+import (
+	"bytes"
+	"text/template"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+
+	crv1 "github.com/grtl/mysql-operator/pkg/apis/cr/v1"
+)
+
+// ObjectFromTemplate executes Go template with given cluster object and
+// parses the result into the object structure.
+func ObjectFromTemplate(cluster *crv1.MySQLCluster, object interface{}, templateFile string) error {
```

Comment:
This should probably be called something like ClusterObjectFromTemplate if this is specifically for templates that need a cluster. Maybe write a more general version that we'd also be able to use with the CRD and ConfigMap?